docs/README.md

```diff
@@ -68,3 +78,24 @@ Changes the selected delivery option to be the one which has the given `delivery
 ```tsx
 selectDeliveryOption('PAC')
 ```
```

Comment:
I believe this info should be divided into different sections: Configuration and Modus Operandi. The `Configuration` section should teach users how to set up the app in their VTEX accounts and/or their store themes. In turn, the `Modus Operandi` section should contain further info regarding the app behavior i.e. how users should properly use it.

Reply:
The app is still under construction, and most probably it will become private.. It is really urgent to update the readme with the warning for now 🙏